src/components/Landing/index.tsx

```diff
@@ -169,29 +161,16 @@ export default function Landing() {
 
                 <div className="hide-mobile">
                   <div style={{ display: 'inline-block' }}>
-                    {v2Enabled ? (
-                      <Link
-                        to={'/create'}
-                        onClick={() => {
-                          window.fathom?.trackGoal('IIYVJKNC', 0)
-                        }}
-                      >
-                        <Button type="primary" size="large">
-                          <Trans>Design your project</Trans>
-                        </Button>
-                      </Link>
-                    ) : (
-                      <Button
-                        type="primary"
-                        size="large"
-                        onClick={() => {
-                          window.fathom?.trackGoal('IIYVJKNC', 0)
-                          scrollToCreate()
-                        }}
-                      >
+                    <Link
+                      to={'/create'}
+                      onClick={() => {
+                        window.fathom?.trackGoal('IIYVJKNC', 0)
+                      }}
+                    >
+                      <Button type="primary" size="large">
                         <Trans>Design your project</Trans>
```

Comment:
I just realised that `Design your project` is inconsistent with the Projects tab (`Create Project`). Might be fine though

Reply:
Yeah thats a good point, I think it's fine tbh. Kinda contextually different. I'm also proposing to change this to `Launch your project` on the homepage, which probably doesn't make as much sense as `Create project` on the Projects page.

Idk, I don't feel too strongly though.

Reply:
Neither do I - happy to leave. just thought i'd raise to have a discussion about it